docs/code/code-install-lang-server.asciidoc

```diff
@@ -1,14 +1,15 @@
 [[code-install-lang-server]]
 == Install language server
 
-*Code* comes with built-in language support for TypeScript. You can install additional languages as a {kib} plugin.
+*Code* comes with built-in language support for TypeScript. You can install additional languages as a {kib} plugin. Plugin's reduce the amount of CPU usage required to run Code. Install only the languages needed for your indexed repositories. 
```

Comment:
the main reason is actually reduce the package size of the kibana distribution.

if distribution size is not a problem, then we could pack all the servers inside it, and to not use to much memory, we could just leave some sever disable in the beginning.